transcom/mymove
# B 19785 add spouse pro gear payment packet integration

## [B-19785](https://www13.v1host.com/USTRANSCOM38/story.mvc/Summary?oidToken=Story%3A946164)

## Summary

This PR updates the payment packet to show the customer's spouse gear entitlement if their spouse has pro gear.

## Verification Steps for Reviewers

These are to be checked by a reviewer.


### How to test

1. Create or use existing move that has spouse pro gear
2. Login as a SC
3. Edit and submit move
4. Refresh page **
5. Click "Download Payment Packet"
6. Verify in packet the spouse entitlement is populated
** or login as TOO and find move

### Backend


## Screenshots
### _Before_
<img width="756" alt="Spouse Gear Before" src="https://github.com/transcom/mymove/assets/146969726/13493bc7-9521-460d-aa12-4523aab77e4a">

### _After_
<img width="750" alt="Screenshot 2024-05-13 at 8 21 25 AM" src="https://github.com/transcom/mymove/assets/146969726/af1e6dce-063c-41ce-b7df-45c92a7f4d4c">
